open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/admin/AdminResource.java

```diff
@@ -0,0 +1,37 @@
+package org.odpi.openmetadata.virtualdataconnector.virtualiser.admin;
+
+import org.odpi.openmetadata.adminservices.OMAGServerAdminServices;
+import org.odpi.openmetadata.adminservices.OMAGServerOperationalServices;
+import org.odpi.openmetadata.adminservices.configuration.properties.CohortConfig;
+import org.odpi.openmetadata.adminservices.configuration.properties.LocalRepositoryConfig;
+import org.odpi.openmetadata.adminservices.configuration.properties.OMAGServerConfig;
+import org.odpi.openmetadata.adminservices.properties.OMAGServerConfigResponse;
+import org.odpi.openmetadata.adminservices.properties.VoidResponse;
+import org.odpi.openmetadata.frameworks.connectors.properties.beans.Connection;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.stereotype.Controller;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * VirtualizerAdminResource provides the spring annotations for the server-side
+ * implementation of the administrative interface for
+ * the Virtualizer
+ */
+
+@RestController
```

Comment:
The virtualizer should not need a RestController as it should run in the OMAG Server Chassis